index.html

```diff
@@ -0,0 +1,89 @@
+                            <input type="search" class="nav__search-input" id="search" name="search"
+                                   placeholder="Поиск по сайту">
+                            <button class="nav__search-btn" type="submit">Найти</button>
+                            <label for="search" class="nav__search-label">
+                            <span class="visually-hidden"></span>
+                            </label>
+                        </form>
+                    </li>
+                    <li class="nav__user-item nav__user-item--login">
+                        <a href="#" class="nav__user-link nav__user-link--login">Войти</a>
+                    </li>
+                    <li class="nav__user-item nav__user-item--compare">
+                        <a href="#" class="nav__user-link nav__user-link--compare">Сравнить</a>
+                    </li>
+                    <li class="nav__user-item nav__user-item--cart">
+                        <a href="#" class="nav__user-link nav__user-link--cart">Корзина</a>
+                    </li>
+                </ul>
+                <ul class="nav__list">
+                    <li class="nav__item nav__item--catalog">
+                        <a href="" class="nav__link nav__link--catalog">Каталог</a>
+                             <!-- <div class="simple-nav__sub-wrap">
+                                <ul class="simple-nav__sub-list">
+                                    <li class="simple-nav__sub-column">
+                                        <a href="#">Виртуальная реальность</a>
+                                        <a href="#">Моноподы для селфи</a>
+                                        <a href="#">Экшн-камеры</a>
+                                    </li>
+                                    <li class="simple-nav__sub-column">
+                                        <a href="#">Фитнес-браслеты</a>
+                                        <a href="#">Умные часы</a>
+                                    </li>
+                                    <li class="simple-nav__sub-column">
+                                        <a href="#">Квадрокоптеры</a>
+                                    </li>
+                                </ul>
+                            </div> -->
+                    </li>
+                    <li class="nav__item nav__item--delivery">
+                        <a href="#" class="nav__link nav__link--delivery">Доставка</a>
+                    </li>
+                    <li class="nav__item nav__item--warranty">
+                        <a href="#" class="nav__link nav__link--warranty">Гарантия</a>
+                    </li>
+                    <li class="nav__item nav__item--contacts">
+                        <a href="#" class="nav__link nav__link--contacts">Контакты</a>
+                    </li>
+                </ul>
+                <!--<div class="simple-nav__sub-wrap--opened">
```

Comment:
I mean you should remove opened class, not all menu. Prepare styles and html and simply remove opened class. For example I can make it with javascript. Logic will be like that : when user hover on root menu element javascript will add opened class to submenu. So styles and this class should be ready to use, not commented

Reply:
Why simple-nav here?

Reply:
because at first I changed all menu-nav to simple-nav)))